config/jobs/kubernetes/cloud-provider-alibaba-cloud/cloud-provider-alibaba-cloud-config.yaml

```diff
@@ -5,11 +5,10 @@ presubmits:
     branches:
     - master
     decorate: true
-    labels:
-      preset-service-account: "true"
+    path_alias: k8s.io/cloud-provider-alibaba-cloud
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190514-6c3cafa-master
+      - image: quay.io/kubermatic/gometalinter:latest
```

Comment:
This is also used in https://github.com/kubernetes/test-infra/blob/2a027c8ac661283f68bae829a94e41e570a94635/config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-presubmits.yaml#L36-L46

And for context: this image is maintained by my employer, so it isn't totally random. :P But let me know you want to change it.